examples/radial_interleafed_example.py

```diff
@@ -0,0 +1,125 @@
+# %% [markdown]
+# # Basic example for reconstruction of radial interleafed sampled data
+# This notebook is meant as a first introduction inot mrpro-based
+# reconstruction of radial interleafed data.
+# %%
+import subprocess
+import tempfile
+from pathlib import Path
+
+import matplotlib.pyplot as plt
+import nibabel as nib
+import numpy as np
+import torch
+
+from mrpro.data import CsmData
+from mrpro.data import IData
+from mrpro.data import SpatialDimension
+from mrpro.data._DcfData import DcfData
+from mrpro.data._KData import KData
+from mrpro.data.traj_calculators._KTrajectoryRadial2D import KTrajectoryRadial2D
+from mrpro.operators import SensitivityOp
+from mrpro.operators._FourierOp import FourierOp
+
+# %%
+
+# flag for conversion and saving nifty files
+# If you wish to save the resulting reconstructed image as a nifty fily, set  NIFTI = "True"
+NIFTI = False
+
+# %% [markdown]
+
+# # Download example data
+# The example data is taken from (https://zenodo.org/records/10669837).
+# It entails data acquired using radial-interleafed sequences written in pulseq.
+# %%
+# Create a temporary directory to save the downloaded files
+data_folder = Path(tempfile.mkdtemp())
+print(f'Data will be saved to: {data_folder}')
+
+# Initialize attempt counter and set maximum number of attempts
+attempt = 1
+max_attempts = 5
+
+# Start the download process
+while attempt <= max_attempts:
+    # Zenodo command for downloading
+    zenodo_cmd = f'zenodo_get 10.5281/zenodo.10664974 -o {str(data_folder)}'
+
+    # Execute the Zenodo command
+    out = subprocess.call(zenodo_cmd, shell=True)
+
+    # Check if the download was successful
+    if out == 0:
+        print('Download successful.')
+        break
+    else:
```

Comment:
Q: why do we need to retry this?

Reply:
Because it fails relatively often. This is probably due to the Zenodo API.

Reply:
ah, thats why there is a retry option for zenodo_get...  :D  (`-r INT`)


also, what is the advantage of subprocess call over 
```import zenodo_get
from pathlib import Path
DATASET = "10669837" # use dataset instead of doi to avoid additional call to resolve doi...
outdir = Path("/tmp/bliblablu")
zenodo_get.zenodo_get([DATASET, "-r", 5, "-o", outdir]) # r: retries
```

Reply:
I will re-write the retry logic to what you posted
The advantge of subprocess I cannot tell you. This code was taken from PTBRecon

Reply:
The I would suggest using the python call of the function.

Reply:
Done, thanks :)